spec/swagger_schemas/requests/participant_declaration_completed_request.rb

```diff
@@ -0,0 +1,70 @@
+PARTICIPANT_DECLARATION_COMPLETED_REQUEST = {
+  description: "An NPQ completed participant declaration",
+  type: :object,
+  required: %i[
+    participant_id
+    declaration_type
+    declaration_date
+    course_identifier
+    has_passed
+  ],
+  additionalProperties: false,
+  properties: {
+    participant_id: {
+      description: "The unique id of the participant",
+      type: :string,
+      required: true,
+      format: :uuid,
+      example: "db3a7848-7308-4879-942a-c4a70ced400a",
+    },
+    declaration_type: {
+      description: "The event declaration type",
+      type: :string,
+      required: true,
+      enum: %w[
+        completed
+      ],
+      example: "completed",
+    },
+    declaration_date: {
+      description: "The event declaration date",
+      type: :string,
+      required: true,
+      format: "date-time",
+      example: "2021-05-31T02:21:32.000Z",
+    },
+    course_identifier: {
+      description: "The type of course the participant is enrolled in",
+      type: :string,
+      required: true,
+      enum: %w[
```

Comment:
could we use `Course::IDENTIFIERS` and `Course::IDENTIFIERS.first` here similar to declarations?